modules/quant_api/src/EventSubscriber/QuantApi.php

```diff
@@ -276,12 +276,12 @@ public function onMedia(QuantFileEvent $event) {
    */
   public function onUnpublish(QuantEvent $event) {
     $url = $event->getLocation();
-
     try {
       $res = $this->client->unpublish($url);
     }
     catch (\Exception $error) {
       $this->logger->error($error->getMessage());
+      return FALSE;
```

Comment:
Was seeing errors in the logs because `$res` is not defined.

Reply:
This one was separated out here a bit ago: https://github.com/quantcdn/drupal/issues/146